InkscapeExtension/ExtendedHatchFill.py

```diff
@@ -96,7 +102,91 @@ def recursivelyTraverseSvg(self, a_node_list):
                     self.interstices((h[0], h[1]), (h[2], h[3]))
 
     def buildHatchLinePath(self, pt1, pt2):
-        return 'M {0:f},{1:f} l {2:f},{3:f} '.format(pt1[0], pt1[1], pt2[0] - pt1[0], pt2[1] - pt1[1])
+        x1, y1 = pt1[0], pt1[1]
+        x2, y2 = pt2[0], pt2[1]
+
+        od = self.options.outer_dist  # Outer contour indent
+        cc = self.options.curv_coef  # Curvature coefficient (steepness angle)
+
+        # absolute length of the segment between the points
+        totLen = ((x2 - x1) ** 2 + (y2 - y1) ** 2) ** 0.5
```

Comment:
The requested change (replace all "x2 - x1" with "w" defined on the line 113, same is for "y2-y1" -> "h" from line 114) is not resolved. Please either make the change or explain why it shouldn't be applied.